packages/kit/src/compatibility.ts

```diff
@@ -108,7 +108,7 @@ export function isNuxt3 (nuxt: Nuxt = useNuxt()) {
  */
 export function getNuxtVersion (nuxt: Nuxt | any = useNuxt() /* TODO: LegacyNuxt */) {
   const rawVersion = nuxt?._version || nuxt?.version || nuxt?.constructor?.version
-  if (!rawVersion) {
+  if (!rawVersion || typeof rawVersion !== 'string') {
```

Comment:
```suggestion
  if (typeof rawVersion !== 'string') {
```

I think we can directly do this

Reply:
I think you're right!